docs/tutorials/pre_executed/custom_search.ipynb

```diff
@@ -0,0 +1,918 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "46a0e317",
+   "metadata": {},
+   "source": [
+    "# Creating a custom search with global statistics\n",
+    "\n",
+    "## Learning Objectives\n",
+    "\n",
+    "In this tutorial, we will:\n",
+    "- showcase additional statistics available on all catalogs\n",
+    "- illustrate how to filter a catalog using those statistics\n",
+    "- demonstrate a custom structured search filter class\n",
+    "\n",
+    "## Introduction\n",
+    "\n",
+    "We're going to find all stars in GAIA Bailor-Jones that are closer than 3 parsecs away, using median photogeometric distances estimates. \n",
```

Comment:
I've never seen the phrase "photogeometric distances", so I have no background here. Is it usually said like so, or would "photogeometric distance estimates" be preferred?